Kentico.Kontent.Delivery/Builders/DeliveryClient/IDeliveryClientBuilder.cs

```diff
@@ -1,5 +1,8 @@
 ﻿using System;
 using System.Net.Http;
+using Kentico.Kontent.Delivery.Abstractions;
```

Comment:
warning on line 13 in this file after the refactoring

Reply:
Fixed